packages/vitest/src/node/stdin.ts

```diff
@@ -95,12 +97,24 @@ export function registerConsoleShortcuts(ctx: Vitest) {
 
   async function inputNamePattern() {
     off()
-    const { filter = '' }: { filter: string } = await prompt([{
-      name: 'filter',
-      type: 'text',
-      message: 'Input test name pattern (RegExp)',
-      initial: ctx.configOverride.testNamePattern?.source || '',
-    }])
+    const watchFilter = new WatchFilter('Input test name pattern (RegExp)')
+    const filter = await watchFilter.filter(async (str: string) => {
+      const files = await ctx.state.getFiles()
+      const tasks = files.map(file => file.tasks).flat()
```

Comment:
You don't really need to do that, you can pass down files to `getTests`

Reply:
@sheremet-va 
Thanks!
I fixed it. (6a0355d)